### DISCUSSION TITLE
How to display the presentation fully unfolded?

### DISCUSSION THREAD

**Discussion by firedev** [marp-team/marp#472] [Q&A] [answered]
Hi,

I would like to convert my presentation into a web page using Marp. Is it possible to export as plain HTML as it looks like in PDF? One page after another. My goal is to display it as a long page of slides in HTML, similar to how it appears in the VSCode preview window.

<img width="586" alt="image" src="https://github.com/marp-team/marp/assets/115612/e8a23603-bda9-42d8-839c-e45d3ed43d89">

**Comment by yhatt** [comment, marked as answer]
Potentially duplicated: Please refer to #445.

**Comment by firedev** [comment]
@yhatt Perfect, thanks
